docs/cmd/kn.md

```diff
@@ -0,0 +1,33 @@
+## kn
```

Comment:
How could this sneak in ? We need to ensure that the documentation generated is only for the plugin itself

Reply:
/hold

looks like `hack/build.sh` needs some tweaks